src/SettingsForm.jsx

```diff
@@ -64,29 +68,34 @@ class UnwrappedEditSettingForm extends React.Component {
           </Form.Item>
           <Form.Item label="Percentage">
             {getFieldDecorator("Percentage", {
-              initialValue: data["advancedroundrobin__Percent"]
+              initialValue: data["advancedroundrobin__Percent"],
             })(<InputNumber />)}
           </Form.Item>
           <Form.Item label="Email For Notifications">
             {getFieldDecorator("email", {
-              initialValue: data.Email
+              initialValue: data.Email,
             })(<Input />)}
           </Form.Item>
           <Form.Item label="Disabled Until (useful for holidays)">
             {getFieldDecorator("Disabled_Until", {
-              initialValue: disabledUntilDate
+              initialValue: disabledUntilDate,
             })(<DatePicker format={"MMM-DD-YYYY"} />)}
           </Form.Item>
           <Form.Item label="Field Criteria">
             {getFieldDecorator("fieldCriteria", {
-              initialValue: data["fieldCriteriaForUI"]
+              initialValue: data["fieldCriteriaForUI"],
             })(
               <FieldCriteriaTable
```

Comment:
Can you work on this table?
1. Add auto-save functionality (same way you did for the availability component)
2. Add remove button to allow users to remove a field criteria row

Reply:
@jeznag sure I will. So the save/edit button will be replaced by the remove button?

Reply:
@jeznag for auto save in field criteria table won't it cause issues? Because right if we add a new row then we need to fill all the values. But with auto save feature we will update the field as soon as it is changed. Also the names of first and second columns combined are saved. So we need both the values to save

Reply:
Correct RE remove. Good point. You can make it only auto save if a field and criteria type have been chosen